src/communication/xcWebSocketSubscriber.ts

```diff
@@ -32,6 +32,24 @@ class Subscriber {
         this.privateTopics = privateTopics;
     }
 
+    getHeartBeat(heartBeatInterval: number) {
```

Comment:
done. In fact, there is an issue to review and fix all the types for subscriber module.